apm-agent-plugins/apm-urlconnection-plugin/src/test/java/co/elastic/apm/agent/urlconnection/HttpUrlConnectionInstrumentationTest.java

```diff
@@ -0,0 +1,369 @@
+package co.elastic.apm.agent.urlconnection;
+
+import co.elastic.apm.agent.httpclient.AbstractHttpClientInstrumentationTest;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.lang.reflect.Field;
+import java.net.Authenticator;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.security.Permission;
+import java.util.List;
+import java.util.Map;
+
+public class HttpUrlConnectionInstrumentationTest extends AbstractHttpClientInstrumentationTest {
+
+    @Override
+    protected void performGet(String path) throws Exception {
+        final HttpURLConnection urlConnection = new HttpURLConnectionWrapper(new HttpURLConnectionWrapper((HttpURLConnection) new URL(path).openConnection()));
```

Comment:
wrapping the wrapper?

Reply:
That tests that wrapping does not create multiple spans